tests/Benchmarks/System.Binary.Base64/Stiching.cs

```diff
@@ -0,0 +1,73 @@
+﻿using BenchmarkDotNet.Attributes;
+using System;
+using System.Binary.Base64Experimental;
+using Base64Decoder = System.Buffers.Text.Base64;
+
+namespace Benchmarks.System.Binary.Base64.Benchmarks
+{
+    public class Stiching
+    {
+        [Params(1000, 5000, 10000, 20000, 50000)]
+        public int InputBufferSize;
```

Comment:
@kant2002 nice combination of using both `Params` and `Arguments`! 👍

Reply:
During that refactoring, I wish that ArgumentSource access mapped Param value for current iteration. That greatly improve "No third call" case.